app/workers/deletes.php

```diff
@@ -563,6 +564,14 @@ protected function deleteBucket(Document $document, string $projectId)
                 $doSpacesAcl = 'private';
                 $device = new DOSpaces(APP_STORAGE_UPLOADS . '/app-' . $projectId, $doSpacesAccessKey, $doSpacesSecretKey, $doSpacesBucket, $doSpacesRegion, $doSpacesAcl);
                 break;
+            case Storage::DEVICE_BACKBLAZE:
```

Comment:
I think we might be able to use the parent method instead of repeating our device init logic in this worker: https://github.com/appwrite/appwrite/blob/master/src/Appwrite/Resque/Worker.php#L262 - you can double check with 
@lohanidamodar to make sure it makes sense.

Reply:
addressed